### PR TITLE
644 filter connections user

### PR DESCRIPTION
**Pull request by tkratky**

### CODE REVIEW

**Comment by tkratky** [comment]
testFilterConnectionsByUser is part of this PR https://github.com/EnMasseProject/enmasse/issues/644

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java [hunk 1]
```diff
@@ -223,6 +219,46 @@ public void doTestFilterConnectionsByEncrypted() throws Exception {
         receivers.forEach(AbstractClient::stop);
     }
 
+    public void doTestFilterConnectionsByUser() throws Exception {
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: Where you stop clients? I see stopClients only in catch.

File: systemtests/src/test/java/io/enmasse/systemtest/TestBase.java [hunk 2]
```diff
@@ -439,6 +445,126 @@ protected int getSubscriberCount(AmqpClient queueClient, Destination replyQueue,
...
+        return attachReceivers(addressSpace, destination, receiverCount, username, password);
+    }
+
+    /**
+     * attach N receivers into one address with own username/password
+     */
+    protected List<AbstractClient> attachReceivers(AddressSpace addressSpace, Destination destination, int receiverCount, String username, String password) throws Exception {
+        ArgumentMap arguments = new ArgumentMap();
+        arguments.put(Argument.BROKER, getRouteEndpoint(addressSpace).toString());
+        arguments.put(Argument.TIMEOUT, "180");
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/TestBase.java]: 180 is too much, default executor timeout for async client is 120, this timeout is like fetch.

Comment (tkratky) [reply systemtests/src/test/java/io/enmasse/systemtest/TestBase.java]: ah, thanks, it was for my debugging purpose -> fixed

File: systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java (hunk 1 above)
Comment (tkratky) [reply systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java]: of course it is my fault, it must be in 'finally' block (thanks again) -> fixed

**Pull request merged**